packages/gatsby-recipes/src/providers/gatsby/plugin.test.js

```diff
@@ -69,7 +81,7 @@ describe(`gatsby-plugin resource`, () => {
   test(`all returns plugins as array`, async () => {
     const result = await plugin.all({ root: STARTER_BLOG_FIXTURE })
 
-    expect(result).toMatchSnapshot()
+    expect(result.map(normalizePlugin)).toMatchSnapshot()
```

Comment:
Sadly jest doesn't work well with array matching but this is the proper way to do it in jest:

```suggestion
    result.forEach(res => {
      expect(res).toMatchSnapshot({
        readme: expect.any(String),
        description: expect.any(String),
      })
    })
```

Reply:
I did consider this approach, but I decided that we want to be checking the smallest amount possible to verify what we're testing (i.e. that it is creating the plugins list properly). Explicitly allowing the props rather than excluding known bad ones means that we're less likely to be bitten by this sort of thing again.